### PR TITLE
Update local dev tips

### PR DESCRIPTION
**Pull request by tdivoll**
This makes it more clear that the Docker tips are specific to using the Cylc local pipeline, whereas the rest is for using the CLI local pipeline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,8 +212,8 @@ docker rm $(docker ps -aq)
 ```bash
 docker image prune
 ```
-
-When running locally, make sure you have at least Julia 1.9.0 installed with the correct architecture for your local machine. (https://julialang.org/downloads/)
+#### CLI local development
```
Comment (danielmwatkins) [inline README.md:215]: Is the acronym CLI defined anywhere?

**Comment by tdivoll** [comment]
Also closes #118

**Review by danielmwatkins — COMMENTED** [review]
Looks good to me!

File: workflow/scripts/ice-floe-tracker.jl
Comment (cpaniaguam) [inline workflow/scripts/ice-floe-tracker.jl:1]: Kind of strange this is showing here. Any idea why?

**Review by cpaniaguam — APPROVED** [review]

File: workflow/scripts/ice-floe-tracker.jl
Comment (tdivoll) [reply workflow/scripts/ice-floe-tracker.jl:1]: I think it's just from changing the permissions on the file. I think this is the change? `100644 → 100755` We weren't able to print the help on that file unless I changed it.

**Pull request merged**